py-base/environment.yml

```
@@ -0,0 +1,86 @@
+name: OHW-py
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9
+  - pangeo-notebook=2022.07.12
```

I'm pretty sure we can bump to 3.10 here. Fingers crosses!

```suggestion
  - python=3.10
  - pangeo-notebook=2022.07.12
```